kotlin-dsl/src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyActionExtensions.kt

```diff
@@ -537,7 +537,7 @@ fun ChutneyStepBuilder.HttpPostAction(
     target: String,
     uri: String,
     headers: Map<String, Any> = mapOf(),
-    body: Any?,
+    body: Any? = null,
```

Comment:
I don't know if it's a good idea to allow a post with no body without explicitly say so at the client side ?